### PR TITLE
Feature/31 implement noaa

### PR DESCRIPTION
**Pull request by JeffHeck**
When you have time, please review.
If all looks good, I think I can use this same technique for the TJs functionality.

### CODE REVIEW

File: app/src/test/java/com/lokico/PSWind/NOAAUnitTests.java [hunk 1]
```diff
@@ -0,0 +1,286 @@
+package com.lokico.PSWind;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by Jeff on 3/31/2016.
```
Comment (loki980) [inline app/src/test/java/com/lokico/PSWind/NOAAUnitTests.java:8]: In general, comments before a function should reflect the content of the function itself.  This is looking like the start of a "changelog".  History of contributions are found in source control, not the source itself.  Changelogs are a collection of issues resolved from your feature/bug tracker.

File: app/src/main/java/com/lokico/PSWind/NOAA.java [hunk 2]
```diff
@@ -85,10 +85,16 @@ static boolean isActivityImplemented() {
     // Convert raw response for a NOAA request
     // Remove unwanted links and text
     static String modifyRawResponse(final String rawHTML) {
```
Comment (loki980) [inline app/src/main/java/com/lokico/PSWind/NOAA.java]: Good move making a new function for this.

File: app/src/main/java/com/lokico/PSWind/NOAA.java [hunk 3]
```diff
@@ -1,17 +1,108 @@
...
     }
 
     static boolean isActivityImplemented() {
-        return false;
+        return true;
+    }
+
+    // Convert raw response for a NOAA request
+    // Remove unwanted links and text
+    static String modifyRawResponse(final String rawHTML) {
```
Comment (loki980) [inline app/src/main/java/com/lokico/PSWind/NOAA.java:87]: Good move making a new function for this.

File: app/src/main/java/com/lokico/PSWind/NOAA.java [hunk 4]
```diff
@@ -1,17 +1,108 @@
...
 
+    private static final String TAG = "NOAA";
+    RequestQueue mRequestQueue;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_noa);
+
+        mRequestQueue = Volley.newRequestQueue(getApplicationContext());
```
Comment (loki980) [inline app/src/main/java/com/lokico/PSWind/NOAA.java:28]: While it's relatively easy to understand, I'm a big stickler for using more comments than you're been using.  It's easier for other people to understand why, for example, you're setting to LOAD_NO_CACHE and enabling javascript if you explain yourself here.  It really helps people reviewing code.

Pretty frequently the "other people" I mentioned is actually "future you".  You'll look at something and ask "Who in their right mind wrote this code.  I have no idea what's going on here."  You go to look up who wrote it to ask them for an explanation ... and you're sad when you figure it out.  =)

File: app/src/test/java/com/lokico/PSWind/NOAAUnitTests.java (hunk 1 above)
Comment (JeffHeck) [inline app/src/test/java/com/lokico/PSWind/NOAAUnitTests.java:8]: Good point. The Android Studio auto wizard added that. I'll clean it up.

File: app/src/main/java/com/lokico/PSWind/NOAA.java (hunk 4 above)
Comment (JeffHeck) [inline app/src/main/java/com/lokico/PSWind/NOAA.java:28]: I agree on adding enough comments so the future me can understand what the old me was thinking.
I have known folks who dislike all comments though and just rely on good variable/method naming.

I'll add an item to my check list to make sure I have commented enough. I'll also make sure a JavaDoc run is complete.

**Pull request merged**